com.unity.multiplayer.mlapi/Runtime/Serialization/NetworkReader.cs

```diff
@@ -41,6 +41,15 @@ public void SetStream(Stream stream)
             m_NetworkSource = stream as NetworkBuffer;
         }
 
+        /// <summary>
+        /// Retrieves the underlying stream the reader is reading from
+        /// </summary>
+        /// <returns></returns>
+        public Stream GetStream()
```

Comment:
Use an auto property maybe?

Reply:
Chose the method here to be consistent with NetworkWriter, which has the same method with the same name.